server/routes/questions.route.js

```diff
@@ -1,5 +1,41 @@
-var express = require('express')
-var app = express()
-var route = express.Router()
+var express = require('express');
+const TOKEN = require('../../Questions/component/config.js');
+const axios = require('axios');
+
+var app = express();
+var route = express.Router();
+const API = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo';
+
+route.get('/qa/questions', (req, res) => {
```

Comment:
@kevin-the-engi  I'm getting internal server error. Maybe the API token I used is outdated? As it stands the server can't find config.js so I had to input the token manually. Made a note to discuss the config.js situation during standup.